### PR TITLE
fix(hardware-testing): 96ch diagnostics tip presence polarity

### PR DESCRIPTION
**Pull request by andySigler**
# Overview

Polarity when reading 96ch tip-presence sensor was switched at some point, and so the production script must also switch to match it.

# Test Plan

# Changelog

# Review requests

# Risk assessment

### CODE REVIEW

File: hardware-testing/hardware_testing/production_qc/ninety_six_assembly_qc_ot3/test_tip_sensor.py [hunk 1]
```diff
@@ -82,14 +82,14 @@ async def run(api: OT3API, report: CSVReport, section: str) -> None:
     slot_5 = helpers_ot3.get_slot_calibration_square_position_ot3(5)
     home_pos = await api.gantry_position(OT3Mount.LEFT)
     await api.move_to(OT3Mount.LEFT, slot_5._replace(z=home_pos.z))
-    for expected_state in [False, True]:
+    for expected_state in [True, False]:
```
Comment (sfoster1) [inline hardware-testing/hardware_testing/production_qc/ninety_six_assembly_qc_ot3/test_tip_sensor.py:85]: Wait if you switch this order but also switch the else/if order below this doesn't really do anything right? You're still expecting it to be `False` with no tips (but now that's second) and `True` with tips (but now that's first)

File: hardware-testing/hardware_testing/production_qc/ninety_six_assembly_qc_ot3/test_droplets.py [hunk 2]
```diff
@@ -29,7 +29,7 @@
 TIP_RACK_96_SLOT = 4
 TIP_RACK_PARTIAL_SLOT = 5
 RESERVOIR_SLOT = 2
-TRASH_SLOT = 1
+TRASH_SLOT = 12
```
Comment (sfoster1) [inline hardware-testing/hardware_testing/production_qc/ninety_six_assembly_qc_ot3/test_droplets.py:32]: where's this one coming from

File: hardware-testing/hardware_testing/production_qc/ninety_six_assembly_qc_ot3/test_tip_sensor.py (hunk 1 above)
Comment (Andiiiiiiyy) [reply hardware-testing/hardware_testing/production_qc/ninety_six_assembly_qc_ot3/test_tip_sensor.py:85]: but actually, the responds of "get tip status" change, so the results will be different,  I want to do the same action to keep the SOP same.

File: hardware-testing/hardware_testing/production_qc/ninety_six_assembly_qc_ot3/test_droplets.py (hunk 2 above)
Comment (andySigler) [reply hardware-testing/hardware_testing/production_qc/ninety_six_assembly_qc_ot3/test_droplets.py:32]: I had put the trash originally in slot 1 for this test, but looks like @Andiiiiiiyy prefers it in slot 12 which is fine by me

File: hardware-testing/hardware_testing/production_qc/ninety_six_assembly_qc_ot3/test_tip_sensor.py (hunk 1 above)
Comment (andySigler) [reply hardware-testing/hardware_testing/production_qc/ninety_six_assembly_qc_ot3/test_tip_sensor.py:85]: looks correct to me @sfoster1 . We are now first expecting `True`, and when it is true that means there are no tips. The ordering of the if/else is arbitrary

Comment (sfoster1) [reply hardware-testing/hardware_testing/production_qc/ninety_six_assembly_qc_ot3/test_tip_sensor.py:85]: Oh. Right. Sorry

**Review by sfoster1 — APPROVED** [review]
Looks great to me!

**Pull request merged**